tests/testthat/test-extract_dials_parameter.R

```diff
@@ -0,0 +1,111 @@
+
+load(test_path("test_object.RData"))
+
+test_that('regular usage', {
+  # perhaps move these to extratests
```

Comment:
is the reason for these tests being commented out that they'd need `tune` to be loaded?